crates/meroctl/src/output.rs

```diff
@@ -41,7 +41,10 @@ pub struct InfoLine<'a>(pub &'a str);
 
 impl Report for InfoLine<'_> {
     fn report(&self) {
-        println!("{} {}", "[INFO]".green(), self.0);
+        let mut table = Table::new();
+        let _ = table.set_header(vec![Cell::new("INFO").fg(comfy_table::Color::Green)]);
+        let _ = table.add_row(vec![self.0]);
+        println!("{table}");
```

Comment:
I don't think we want a table here since this is a way to show unstructured data (that already doesn't have a JSON representation). The same applies to `ErrorLine`, but before you change it, let's see some screenshots.

cc: @fbozic

Reply:
I'll send them once #1239 gets merged. I was trying to set it up and kept getting that error but I never they were outdated.

Or maybe if you can test it on your end

Reply:
@Nathy-bajo #1239 will only cover 2 help commands which had additional "--" in help table fyi.

Reply:
@Nathy-bajo, unsure if you linked the right PR, but that's merged now

Reply:
@Nathy-bajo, this is still unresolved, let's not use a table view for `InfoLine`, `ErrorLine` and `WarnLine`

they are specifically for unstructured outputs. everything else is for representing structured data.